packages/prisma/seed-app-store.ts

```diff
@@ -86,6 +87,13 @@ async function main() {
     });
   }
   await createApp("space-booking", "spacebooking", ["other"], "spacebooking_other");
+  await createApp("vital-automation", "vital", ["other"], "vital_other", {
+    mode: process.env.VITAL_DEVELOPMENT_MODE || "sandbox",
+    region: process.env.VITAL_REGION || "us",
+    client_id: process.env.VITAL_CLIENT_ID,
```

Comment:
Need to migratw this to api-key

Reply:
So that's why I couldn't follow the readme instruction. 😅